mmv1/third_party/terraform/services/storage/resource_storage_bucket.go.tmpl

```diff
@@ -549,6 +549,23 @@ func ResourceStorageBucket() *schema.Resource {
 					},
 				},
 			},
+			"hierarchical_namespace": {
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Optional:    true,
```

Comment:
We'll want to add a DiffSuppressFunc to this top-level field similar to this one: https://github.com/hashicorp/terraform-provider-google-beta/blob/804148d2559f8c6628258cfa232cd81c72ed6233/google-beta/services/container/resource_container_cluster.go#L6814-L6826

We're always storing the full ("hydrated") value in state in our flatten function where null is mapped to an item being present with `enabled = false`, so if we see a diff from `.#` (length) of 1 -> 0 and `enabled`  is `false`, we can return `true` from the function to suppress the diff.

Reply:
added the suggested change for DiffSuppress func